chef/cookbooks/bind9/recipes/default.rb

```diff
@@ -403,6 +403,7 @@ def make_zone(zone)
   command "nscd -i hosts"
   action :nothing
   subscribes :run, "template[/etc/bind/db.#{node[:dns][:domain]}]"
+  only_if { File.exist?("/var/run/nscd/nscd.pid") }
```

Comment:
Are we sure that path is correct for all systems? Or would `only_if "which nscd > /dev/null"` be better?